nengo_gui/components/netgraph.py

```diff
@@ -469,8 +470,12 @@ def get_extra_info(self, obj):
         elif isinstance(obj, nengo.Ensemble):
             info['dimensions'] = int(obj.size_out)
             info['n_neurons'] = int(obj.n_neurons)
+        elif isinstance(obj, spa.Compare):
+            info['scalar_out'] = True
```

Comment:
I'm okay with this special-case code for now, but we should add a note here that if we start having a lot of cases added, we should make this more generic.  For example, we could just do the same check as in value.py:

```
elif ((isinstance(obj, spa.module.Module) and obj.outputs.haskey('default')) or
      (isinstance(obj, spa.Network) and hasattr(obj, 'output'))):
             info['has_default_output'] = True
```

Hmm, and I'd also say that ```has_default_output``` may be clearer than ```scalar_out```, especially since there's nothing in this version of the code that forces it to be a scalar.....